mslib/mscolab/mscolab.py

```diff
@@ -93,6 +96,264 @@ def handle_db_seed():
     print("Database seeded successfully!")
 
 
+def handle_mscolab_certificate_init():
+    print('generating CRTs for the mscolab server......')
```

Comment:
There are some print-function calls in here already, but we should generally use the logging for console output.